scripts/pc2sandbox.sh

```diff
@@ -65,16 +82,35 @@ timelimit, in seconds
 SAGE
 }
 
+# Function to kill all processes in the cgroupv2 
+KillcgroupProcs()
+{
+	if test -n ${PC2_SANDBOX_CGROUP_PATH_KILL}
+	then
+		DEBUG echo "Purging cgroup ${PC2_SANDBOX_CGROUP_PATH_KILL} of processes"
+		echo 1 > ${PC2_SANDBOX_CGROUP_PATH_KILL}
+	fi
+}
+
+# Kill active children and stragglers
+KillChildProcs()
+{
+	DEBUG echo "Killing off submission process group $submissionpid and all children"
+	# Kill off process group
+	if test -n "$submissionpid"
+	then
+		kill -9 -$submissionpid
```

Comment:
again with - before the pid,  I've looked at bash kill and /bin/kill and can find no reference for anything but the -1 case to kill 
```
 it indicates all processes except the kill  process  itself and init.
```

Reply:
I would rather all those be just written as -1 if that's the case.

Reply:
I suppose, but i thought that killing the pgroup was safer, just in case it was accidently run by root.  Doesn't kill -9 -1 kill everyone in that case?  Just killing the PGRP is safer, I would think.